app/controllers/studios_controller.rb

```diff
@@ -20,6 +21,12 @@ def destroy
     redirect_to store_path(@store), danger: 'スタジオを削除しました'
   end
 
+  def show
+    @studio = Studio.find(params[:id])
+    @today = Date.today
+    @reservations = Reservation.where(studio_id:@studio.id).where(date:@today..@today.since(6.days))
```

Comment:
`Reservation.where(studio_id:@studio.id).where(date:@today..@today.since(6.days))` を `Reservation.where(studio_id: @studio, date: today..today.since(6.days))` に修正。

* `@studio.id` は `@studio`
* `:` のあとはかならず半角スペース
* `@today` は変数が宣言されたメソッドの中でしか使わなにでローカル変数に修正する。
* whereメソッドは1つのセンテンスで2度使わない。複数の条件で取得したい場合は `,` を使う。